community/consistency-check/src/test/java/org/neo4j/consistency/checking/full/FullCheckIntegrationTest.java

```diff
@@ -677,7 +681,7 @@ protected void transactionData( GraphStoreFixture.TransactionDataBuilder tx,
 
         // then
         on( stats ).verify( RecordType.RELATIONSHIP, 2 )
-                   .verify( RecordType.COUNTS, 1 )
+                   .verify( RecordType.COUNTS, 3 )
```

Comment:
Because some of those counts updates were inferred from the `RelationshipCommand`.

With this change to require specific commands for updating counts, I didn't feel the need to add the corresponding counts update commands to this test, since it was testing for the absence of such commands anyhow.